src/net.h

```diff
@@ -361,6 +362,16 @@ class CConnman
     void ThreadSocketHandler();
     void ThreadDNSAddressSeed();
 
+    // Reachable nodes should periodically make requests
+    // to DNS servers even if they don't need them.
+    // Then, when new nodes from the same subnets need to query DNS,
+    // they are likely to hit the cache instead.
+    // For non-reachable nodes this means more privacy,
+    // because fewer actors would know about their query.
+    // For reachable nodes there is no privacy loss due to this,
+    // because being reachable already makes them public.
+    void DNSCachesUpdate();
```

Comment:
nit: Both of the newly added methods can be `const` because they are read-only, no?
```suggestion
    void DNSCachesUpdate() const;
```